tests/Http/Controllers/CelebTest.php

```diff
@@ -2,175 +2,414 @@
+            ->assertRedirect();
     }
 
     /** @test */
     public function guest_cannot_see_celebs_edit_view()
     {
-        $celeb = \App\Models\Celeb::factory()->create();
+        $celeb = Celeb::factory()->create();
         $this->getJson("celebs/$celeb->id/edit")
-            ->assertStatus(403);
+            ->assertRedirect();
     }
 
     /** @test */
-    public function admin_can_access_celebs_update()
+    public function admin_can_update_a_celeb()
     {
-        $celeb = \App\Models\Celeb::factory()->create();
-        $admin = \App\Models\User::factory()->create([
-            'is_admin' => true
+        $celeb = Celeb::factory()->create([
+            'name' => 'Jennifer Fallinbury',
+            'date_of_birth' => Carbon::now()->subYears(30),
+            'photo' => 'https://generic/photo1_700x600'
         ]);
+        $admin = User::factory()->admin()->create();
 
         $this->actingAs($admin)
-            ->putJson("celebs/$celeb->id")
-            ->assertOk();
+            ->put("celebs/$celeb->id", [
+                'name' => 'Jennifer Jaxon',
+                'date_of_birth' => Carbon::now()->subYears(30),
+                'photo' => 'https://generic/photo1_700x600'
+            ])->assertSessionHasNoErrors();
+
+        $this->assertDatabaseMissing('celebs', [
+            'id' => $celeb->id,
+            'name' => 'Jennifer Fallinbury',
+            'date_of_birth' => Carbon::now()->subYears(30),
+            'photo' => 'https://generic/photo1_700x600'
+        ]);
+
+        $this->assertDatabaseHas('celebs', [
+            'id' => $celeb->id,
+            'name' => 'Jennifer Jaxon',
+            'date_of_birth' => Carbon::now()->subYears(30),
+            'photo' => 'https://generic/photo1_700x600'
+        ]);
     }
 
     /** @test */
-    public function normal_user_cannot_access_celebs_update()
+    public function user_cannot_update_celeb()
     {
-        $celeb = \App\Models\Celeb::factory()->create();
-        $user = \App\Models\User::factory()->create([
+        $celeb = Celeb::factory()->create([
+            'name' => 'Joe Generic',
+            'date_of_birth' => Carbon::now()->subYears(30),
+            'photo' => 'https://genericPhotos.com'
+        ]);
+        $user = User::factory()->create([
             'is_admin' => false
         ]);
 
         $this->actingAs($user)
-            ->putJson("celebs/$celeb->id")
-            ->assertStatus(403);
+            ->putJson("celebs/$celeb->id", [
+                'id' => $celeb->id,
+                'name' => 'Joe Invalid',
+                'date_of_birth' => Carbon::now()->subYears(30),
+                'photo' => 'https://genericPhotos.com'
+            ])->assertRedirect();
+
+        $this->assertDatabaseMissing('celebs', [
+            'id' => $celeb->id,
+            'name' => 'Joe Invalid',
+            'date_of_birth' => Carbon::now()->subYears(30),
+            'photo' => 'https://genericPhotos.com'
+        ]);
```

Comment:
You can also just leave the one below, if we are checking that the DB has the original one, we do not need to check if the record you send is missing 😄